web/themes/custom/vote_gov/package-lock.json

```diff
@@ -10,297 +10,33 @@
       "license": "ISC",
       "dependencies": {
         "@uswds/uswds": "^3.4.1",
-        "a11y-tabs": "^0.1.2"
+        "a11y-tabs": "^0.1.2",
+        "stylelint": "^15.6.3"
```

Comment:
This package-json needs to be receated. It should not contain an update to stylelint.